cmd/init.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"os"
+	"strings"
+)
+
+type Config struct {
+	ContainerSubsystem string `yaml:"container_subsystem"`
+	Cli                string `yaml:"cli"`
+	OCMUrl             string `yaml:"ocm_url"`
+	OCMUser            string `yaml:"ocm_user"`
+	DefaultVeleroNs    string `yaml:"default_velero_ns"`
+	OfflineAccessToken string `yaml:"offline_access_token"`
+	OpsUtilsDir        string `yaml:"ops_utils_dir"`
+	OpsUtilsDirRw      bool   `yaml:"ops_utils_dir_rw"`
+}
+
+func makeConfig() Config {
+	return Config{
+		ContainerSubsystem: "podman",
+		Cli:                "ocm",
+		DefaultVeleroNs:    "openshift-velero",
+	}
+}
+
+const (
+	OCMUsernamePrompt        = `Provide your ocm user name`
+	OfflineAccessTokenPrompt = `Provide your OCM Offline Access Token from https://cloud.redhat.com/openshift/token`
+	OpsUtilsDirPrompt        = `(Optional) Provide your ops-sop/v4/utils directory.
+This is an absolute path to any necessary scripts you wish to have automatically mounted into your container.
+This is mounted in the "/root/sop_utils" directory in the container.`
+	OpsUtilsDirRwPrompt = `Would you like the ops-sop directory to be mounted as readonly? [y/N]`
+	WaitingForUserInput = `: `
+)
+
+func NewInitCmd() *cobra.Command {
+	var initCmd = &cobra.Command{
+		Use:   "init",
+		Short: "Initializes OCM container configuration",
+		Long: `init will create a config file at ~/.config/ocm-container/config.yaml.
+If a config.yaml file already exists, you will be asked if you want to overwrite it or exit out.`,
+		Run: setupConfig,
+	}
+	return initCmd
+}
+
+func setupConfig(*cobra.Command, []string) {
+	reader := bufio.NewReader(os.Stdin)
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
+	}
+	configPath := userHomeDir + "/.config/ocm-container/config.yaml"
```

Comment:
If we don't go the viper route, we can `import github.com/openshift/occ/pkg/config` and then here we can `configPath := fmt.Sprintf("%s/config.yaml", config.DefaultConfigFileLocation)`